test/core/crypto/vrf/vrf_provider_test.cpp

```diff
@@ -6,14 +6,14 @@
 #include "crypto/vrf/vrf_provider_impl.hpp"
 
 #include <gtest/gtest.h>
-#include "crypto/random_generator/boost_generator.hpp"
 #include "common/mp_utils.hpp"
+#include "crypto/random_generator/boost_generator.hpp"
 
 using kagome::common::Buffer;
 using kagome::crypto::BoostRandomGenerator;
 using kagome::crypto::Sr25519Keypair;
-using kagome::crypto::VRFProviderImpl;
 using kagome::crypto::VRFPreOutput;
+using kagome::crypto::VRFProviderImpl;
 using kagome::crypto::VRFThreshold;
 
 class VRFProviderTest : public testing::Test {
```

Comment:
Are the new methods for transcripts tested? I can see only changes in the existing tests, that test message buffers.